docs/flashbots-core/overview.md

```diff
@@ -0,0 +1,147 @@
+---
+title: overview
+---
+
+Flashbots Core is a permissionless, transparent, and fair ecosystem for frontrunning protection and efficient MEV extraction which preserves the ideals of Ethereum. Flashbots Core provides a private communication channel between Ethereum users and miners for efficiently communicating preferred transaction order within a block.
```

Comment:
since we're not fully permissionless I'd consider maybe changing the language slightly? (eg. on its way to be, aiming to be)